_locales/it/messages.json

```diff
@@ -0,0 +1,300 @@
+	},
+
+	"openFromBookmark": {
+		"message": "Apri da un segnalibro"
+	},
+
+	"openFromLink": {
+		"message": "Apri da un link"
+	},
+
+	"openFromPage": {
+		"message": "Apri da una pagina"
+	},
+
+	"openFromSelection": {
+		"message": "Apri da una selezione"
+	},
+
+	"openFromSidebar": {
+		"message": "Apri dalla barra laterale "
+	},
+
+	"openFromTab": {
+		"message": "Apri da una scheda"
+	},
+
+	"openOptions": {
+		"message": "Apri le opzioni"
+	},
+
+	"openThisAudio": {
+		"message": "Apri questo audio"
+	},
+
+	"openThisBookmark": {
+		"message": "Apri questo segnalibro"
+	},
+
+	"openThisImage": {
+		"message": "Apri questa immagine"
+	},
+
+	"openThisLink": {
+		"message": "Apri questo link"
+	},
+
+	"openThisPage": {
+		"message": "Apri questa pagina"
+	},
+
+	"openThisSelection": {
+		"message": "Apri questa selezione"
+	},
+
+	"openThisSidebar": {
+		"message": "Apri questa barra laterale"
+	},
+
+	"openThisTab": {
+		"message": "Apri questa scheda"
+	},
+
+	"openThisVideo": {
+		"message": "Apri questo video"
+	},
+
+	"openTutorial": {
+		"message": "Apri la guida"
+	},
+
+	"optionsHTMLTitle": {
+		"message": "Optioni",
+		"description": "The title of options.html."
+	},
+
+	"optionsUserAgentHTMLInformation": {
+		"message": "Questa funzionalità richiede il permesso di accedere ai dati dei siti web."
+	},
+
+	"optionsUserAgentHTMLCaution": {
+		"message": "Questo sovrascrive l'user agent ed influisce sulle pagine aperte nella barra laterale e relative estensioni."
+	},
+
+	"placement": {
+		"message": "Comportamento"
+	},
+
+	"placementAllCaution": {
+		"message": "Non ha effetto su barre laterali aperte con modalità 'Scheda' o 'Finestra' e viene sovrascritta da entrambe."
+	},
+
+	"placementAllDescription": {
+		"message": "Reimposta tutte le barre laterali."
+	},
+
+	"placementTabCaution": {
+		"message": "La modalità 'Scheda' può essere sovrascritta solo da se stessa."
+	},
+
+	"placementTabDescription": {
+		"message": "Reimposta la barra laterale sulla scheda corrente."
+	},
+
+	"placementWindowCaution": {
+		"message": "Non ha effetto su barre laterali aperte con modalità 'Scheda' e viene sovrascritta da se stessa o da 'Scheda'."
+	},
+
+	"placementWindowDescription": {
+		"message": "Reimposta le barre laterali sulla finestra corrente."
+	},
+
+	"protocol": {
+		"message": "Protocollo"
+	},
+
+	"reload": {
+		"message": "Ricarica"
+	},
+
+	"specify": {
+		"message": "Specifica"
+	},
+
+	"sidebarHTMLTitle": {
+		"message": "Guida",
+		"description": "The title of sidebar.html."
+	},
+
+	"synchronization": {
+		"message": "Sincronizzazione (account Mozilla)"
+	},
+
+	"tab": {
+		"message": "Scheda"
+	},
+
+	"target": {
+		"message": "Voci target menu"
+	},
+
+	"thisUrlIsNotAValidUrl": {
+		"message": "Questo non è un url valido."
+	},
+
+	"toggle": {
+		"message": "Apri/Chiudi"
+	},
+
+	"toolbarIconAction": {
+		"message": "Azione dell'icona sulla barra degli strumenti"
+	},
```

Comment:
Google Translation suggests "Azione dell icona sulla barra degli strumenti" I have no idea that this suggestion is correct or not. If this translation is intended, please just resolve this comment.